src/css/object/project/process.scss

```
@@ -0,0 +1,104 @@
+@use '../../object/utility/utility.scss' as ut;
+// ページ後半の「転職までの流れ」に関するスタイル
+.p-processo {
```

タイポしちゃってるので修正お願いします！

```suggestion
.p-process {
```

---

今回の記述で"processo"と書いている箇所を全て"process"に修正します。